releasenotes/notes/tphi-757e15fcb24219f9.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+features:
+  - |
+    New experiment Tphi. This is a composite experiment with sub-experiments
```

Comment:
Please add a couple more sentences, including a link to the API docs (I think @ItamarGoldman did it correctly in #365)

Reply:
Done.

Reply:
I updated the setting of `options` above. I added links in the release notes, like @ItamarGoldman  did.